app/src/main/java/com/epfl/drawyourpath/map/MapFragment.kt

```diff
@@ -81,24 +86,33 @@ class MapFragment(private val focusedOnPosition: Boolean = true, private val pat
      * @param location where we want to move the camera
      * @param zoom apply to the camera
      */
-    private fun moveCameraToPosition(location: LatLng, zoom: Float) {
+    private fun moveCameraToPosition(location: LatLng, zoom: Float = DEFAULT_ZOOM) {
         map?.moveCamera(
             CameraUpdateFactory.newLatLngZoom(
                 LatLng(
                     location.latitude,
                     location.longitude,
                 ),
-                DEFAULT_ZOOM,
+                zoom,
             ),
         )
     }
 
+    private fun setupDrawingOnMap(map: GoogleMap) {
```

Comment:
Maybe add comment to explain the role of this function/for example : This function is used to display the run when the user is currently drawing...).